testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -247,6 +247,14 @@ private void addStorageOptions(List<String> commands) {
         String mapStorageProfile = System.getProperty(AUTH_SERVER_QUARKUS_MAP_STORAGE_PROFILE);
 
         if (mapStorageProfile != null) {
+            // We need to drop optimized flag because --storage is build option therefore startup requires re-augmentation
+            commands.removeIf("--optimized"::equals);
+
+            // As config is re-augmented on startup we need to also add --http-relative-path as ant build from
+            // integration-arquillian/servers/auth-server/quarkus/ant/configure.xml is replaced by build invoked on
+            // startup when we add new build option below
+            commands.add("--http-relative-path=/auth");
+
             switch (mapStorageProfile) {
                 case "chm":
                     commands.add("--storage=" + mapStorageProfile);
```

Comment:
@pedroigor This would be possible. However, I thought the build is happening only if some change in config was made, not every time. See https://github.com/keycloak/keycloak/blob/e3af0610e21961c1e8d074a223f9fffee0e56dde/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java#L125